### ISSUE TITLE
Ruby warning email parser that a character class has a duplicated range.

### ISSUE THREAD

**Issue by karloscarweber** [danini-the-panini/kdl-rb#16]
Hi Friend,

While Running tests on **Camping**, which uses **kdl**, Ruby issues a warning that the email parser has a character class with a duplicated range. I think it's a regex error. If you don't get to this I'd like to open a PR to maybe fix it.

Relevate error:
```
/Users/kow/.rvm/gems/ruby-3.3.0/gems/kdl-1.0.5/lib/kdl/types/email/parser.rb:74: warning: character class has duplicated range: /[^\x00-\x1F\s".@]/
```

**Comment by danini-the-panini** [comment]
weird, I can't seem to get the warning to show on my side.

it looks like most of `\s` is already in `\x00-\x1f`, so I'll just replace `\s` with ` ` and that should fix it

**Comment by danini-the-panini** [comment]
@karloscarweber can you check out #17 and see if it makes the warning go away without any adverse issues?

**Comment by karloscarweber** [comment]
That fixes it.

I get the error when installing camping 3.2.3, `gem install camping`, running the camping new command: `camping new myapp`, installing gems via bundler: `bundle install` then running the almost empty test suite: `bundle exec rake`.

Thanks for fixing this so quickly.

**Comment by danini-the-panini** [comment]
Fixed in v1.0.6 release
